packages/kbn-test/src/functional_test_runner/lib/config/schema.ts

```diff
@@ -187,6 +187,15 @@ export const schema = Joi.object()
         sourceArgs: Joi.array(),
         serverArgs: Joi.array(),
         installDir: Joi.string(),
+        /** Options for how FTR should execute and interact with Kibana */
+        runOptions: Joi.object()
+          .keys({
+            /** Log message to wait for before initiating tests, defaults to waiting for Kibana status to be `available` */
+            wait: Joi.object()
+              .regex()
+              .default(/Kibana is now available/),
```

Comment:
> Adds a new kbnTestServer.runOptions.wait config to FTR to allow test suites to customize which log message to wait for before starting tests

We might need to document that plugins might need to adjust the logging level in the tests. The default log level is `info`, but most of the plugins log with the `debug` level.